examples/resource-method-typing/resource_method_typing.bal

```diff
@@ -9,11 +9,10 @@ listener http:Listener h = new (9090);
 
 service /calc on h {
     // Resource method arguments can use user-defined types.
-    // Annotations can be used to refine the mapping between 
-    // Ballerina-declared type and wire format.
+    // Annotations can be used to refine the mapping between Ballerina-declared type and wire
```

Comment:
```suggestion
    // Annotations can be used to refine the mapping between the Ballerina-declared type and wire
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/2930/commits/850aaadbbb3db21491d1ca85c58cf87a6dc589e8.